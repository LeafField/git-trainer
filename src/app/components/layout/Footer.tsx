import React from "react";

const Footer = () => {
  return (
    <footer className="pb-10">
      <small className="block text-center text-white">
        Copyright Â© 2023 LeafField All rights reserved.
      </small>
    </footer>
  );
};

export default Footer;
