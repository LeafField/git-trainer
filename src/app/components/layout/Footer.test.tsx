import { screen, render } from "@testing-library/react";
import Footer from "./Footer";

test("Footerが正しくレンダリングされているか", () => {
  const { container } = render(<Footer />);
  expect(
    screen.getByText("Copyright © 2023 LeafField All rights reserved.")
  ).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
