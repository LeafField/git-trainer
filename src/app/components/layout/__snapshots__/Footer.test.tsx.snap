// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footerが正しくレンダリングされているか 1`] = `
<div>
  <footer
    class="pb-10"
  >
    <small
      class="text-white block text-center"
    >
      Copyright © 2023 LeafField All rights reserved.
    </small>
  </footer>
</div>
`;
